plugins/mcreator-localization/help/fr_FR/biome/raining_possibility.md

```diff
@@ -1 +1,6 @@
-Ce paramètre contrôle la possibilité de pleuvoir à l'intérieur du biome.
\ No newline at end of file
+Ce paramètre contrôle la possibilité de pleuvoir à l'intérieur du biome.
+
+Ce paramètre contrôle également le placement du biome.
+Les biomes avec une possibilité de pleuvoir similaire se placeront plus proches les uns des autres.
+
+Les biomes avec la même possibilité de pleuvoir seront en concurrence pour la même place dans le monde lors de la génération.
```

Comment:
```suggestion
Les biomes avec la même possibilité de pluie seront en concurrence pour la même place dans le monde lors de la génération.
```

Reply:
No need to re-request a demand after. French tips are good.

Reply:
```suggestion
Les biomes avec la même probabilité de pluie seront en concurrence pour la même place dans le monde lors de la génération.
```
You should replace `possibilité` by `probabilité`

Reply:
@Defeatomizer you did not take my request before merging why?

Reply:
I did, but Goldorion suggested changes first. Pretty weird logic, yeah.